README.md

```diff
@@ -100,137 +45,118 @@ For more information on the importance of a professional README for your reposit
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
+- Transactions: Users can create transactions, which are associated with a category. The category can be anything, such as "Food," "Entertainment," or "Transportation."
 
-## 🚀 Live Demo <a name="live-demo"></a>
+- Categories: Users can create and manage categories. This allows them to track their expenses more granularly.
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
```

Comment:
- [ ] [REQUIRED] Good job so far! Please `re-create` `Live Demo` section to your `README` file with `Live link` because you should deploy the application using `Render`, `Netlify` or `GitHub Pages` and `project` should be deployed properly, the `home` and `detail` page works as expected. A `link` to the deployed project should be present in the `README` 🙏🏼 

![image](https://github.com/Ridwanullahi-code/BudgetApp/assets/63915024/31991f98-3b5a-448a-906e-352e9286b327)